docs/examples/viz_multi_screen.py

```diff
@@ -65,16 +65,14 @@
 scene1.add(sphere_actor1)
 scene2.add(sphere_actor2)
 
+interactive = True
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
+if "INTERACTIVE_TUTORIALS" in os.environ:
+    interactive = os.environ["INTERACTIVE_TUTORIALS"].lower() == "true"
```

Comment:
we want the user to avoid that. Can you handle it in the `ShowManager` when you call `show` or `start`

Reply:
Yes, I can create an easy way.